### PR TITLE
Adds temporary default for storageType

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
Me and @calvix decided default was host path, see:
https://gigantic.slack.com/archives/C03CPNRTS/p1505902710000105

I'm thinking we set this as a crappy default for now, get to updating the TPOs next, and then remove it. Sound good?

**Comment by JosephSalisbury** [comment]
I prefer this approach _slightly_ to what I had before, hence the new PR.

File: service/resource/deployment/master_deployment.go [hunk 1]
```diff
@@ -22,8 +22,17 @@ func newMasterDeployments(customObject kvmtpr.CustomObject) ([]*extensionsv1.Dep
 	for i, masterNode := range customObject.Spec.Cluster.Masters {
 		capabilities := customObject.Spec.KVM.Masters[i]
 
+		storageType := key.StorageType(customObject)
+
+		// During migration, some TPOs do not have storage type set.
+		// This specifies a default, until all TPOs have the correct storage type set.
+		// tl;dr - this shouldn't be here. If all TPOs have storageType, remove it.
+		if storageType == "" {
```
Comment (xh3b4sd) [inline service/resource/deployment/master_deployment.go]: Please create an issue for this.

Comment (JosephSalisbury) [reply service/resource/deployment/master_deployment.go]: See https://github.com/giantswarm/giantswarm/issues/1872

**Pull request merged**